disable connectivity map if tile-lazy-loading enabled (#6208)
valhalla/valhalla: CHANGELOG.md

## UNRELEASED
* **Removed**
* **Bug Fix**

>    * FIXED: Disable connectivity-map feature when `tile_url` tile source is set and no tar-extract exists. `GetTileSet` returns no tiles (due to empty `tile_dir` on startup) for this case, so the connectivity map initializes empty and stays empty for its entire lifecycle.

Reviewer: ```suggestion
   * FIXED: Disable connectivity-map when tile source is lazy loading HTTP (from tiles or tar)
```

Reviewer: also look at the other entries pls. missing the PR URL
